.github/workflows/manubot.yaml

```diff
@@ -0,0 +1,40 @@
+name: Manubot
```

Comment:
Maybe this could have a more descriptive name, since the user's might theoretically poke around and see it?

![image](https://user-images.githubusercontent.com/8326331/72445761-415a4d00-3780-11ea-8ff8-e52366ae5943.png)

Maybe `Manubot Continuous Integration` or `Manubot CI` or just `Manubot Build and Deploy Manuscript`

Maybe the file name could be `manubot-ci.yml` or `manubot-build-and-deploy.yml`

Finally, is "manubot" necessary in these names since we're already in the context of manubot, being in a manubot repository.

Reply:
> is "manubot" necessary in these names since we're already in the context of manubot

I am thinking about the situation where someone creates a repository with a manuscript as well as other stuff. The other stuff may need additional workflows. Those other workflows may have nothing to do with Manubot... for example perhaps one runs an R analysis, another one spellchecks pull requests, another makes sure PR authors have signed a contributor license agreement, another one automatically merges approved PRs. In these cases, I envision you want workflow names that best differentiate the specific workflow.

Since the name "Manubot" is used in the context of a workflow, I think adding `CI` or "Continuous Integration" afterwards is a bit redundant. My understanding is that workflows are basically CI, and perhaps GitHub wanted to get away from the CI terminology because it's a bit confusing.

One final point is that I see the GitHub action workflow as become the main way people run manubot. The rest of the manuscript repo will be their content / config files. The action is actually "the manubot", or at least the place where "the manubot" runs.